pkg/noderesourcetopology/cache/attr_watch.go

```diff
@@ -72,7 +73,7 @@ func (wt Watcher) ProcessEvent(ev watch.Event) bool {
 
 	nrtObj, ok := ev.Object.(*topologyv1alpha2.NodeResourceTopology)
 	if !ok {
-		wt.lh.Info("unexpected object %T", ev.Object)
+		wt.lh.Info(fmt.Sprintf("unexpected object %T", ev.Object))
```

Comment:
ditto, remove fmt.XYZ

Reply:
I suggest this approach as bellow.  🤔 

`wt.lh.Info("unexpected object", "kind", fmt.Sprintf("%T", ev.Object))`

Because we use with original writing and will get logcheck output similar to the following

```bash
/home/ubuntu/scheduler-plugins/pkg/noderesourcetopology/cache/attr_watch.go:75:3: logging function "Info" should not use format specifier "%T"
/home/ubuntu/scheduler-plugins/pkg/noderesourcetopology/cache/attr_watch.go:81:43: Key positional arguments are expected to be inlined constant strings. Please replace &{logging KeyNode} provided with string value.
/home/ubuntu/scheduler-plugins/pkg/noderesourcetopology/cache/attr_watch.go:89:38: Key positional arguments are expected to be inlined constant strings. Please replace &{logging KeyNode} provided with string value.
/home/ubuntu/scheduler-plugins/pkg/noderesourcetopology/cache/discardreserved.go:82:33: Key positional arguments are expected to be inlined constant strings. Please replace &{logging KeyPod} provided with string value.
/home/ubuntu/scheduler-plugins/pkg/noderesourcetopology/cache/discardreserved.go:82:65: Key positional arguments are expected to be inlined constant strings. Please replace &{logging KeyPodUID} provided with string value.
```

Reply:
sounds good given `%T` is a bit special here - we cannot interpret it easily w/o using `%`